libbeat/docs/config-file-format.asciidoc

```diff
@@ -238,10 +238,10 @@ Example referencing `es.host` in `output.elasticsearch.hosts`:
 
 ["source","yaml",subs="attributes"]
 ------------------------------------------------------------------------------
-es.host: '$\{ES_HOST:localhost}'
+es.host: '${ES_HOST:localhost}'
 
 output.elasticsearch:
-  hosts: ['http://$\{es.host}:9200']
+  hosts: ['http://${es.host}:9200']
```

Comment:
@nik9000 Here's another place where asciidoctor builds, but asciidoc fails. The erorr is:

```
12:20:15 INFO:build_docs:asciidoc: WARNING: config-file-format.asciidoc: line 244: illegal system attribute 
```

I'm OK with ignoring the CI for now since the CI will run on conf.yaml. WDYT?